cub/cub/agent/agent_radix_sort_histogram.cuh

```diff
@@ -210,7 +207,9 @@ struct AgentRadixSortHistogram
 #pragma unroll
     for (int current_bit = begin_bit, pass = 0; current_bit < end_bit; current_bit += RADIX_BITS, ++pass)
     {
-      int num_bits = CUB_MIN(RADIX_BITS, end_bit - current_bit);
+      // FIXME(bgruber): the following replacement changes SASS for cub.test.device_radix_sort_pairs.lid_0
+      // int num_bits = _CUDA_VSTD::min(+RADIX_BITS, end_bit - current_bit);
+      int num_bits = CUB_MIN(+RADIX_BITS, end_bit - current_bit);
```

Comment:
I am really curious about why we need `+RADIX_BITS`, it should already be an `int` and so are `end_bit` and `current_bit`

What does this do?
```suggestion
      int num_bits = _CUDA_VSTD::min<int>(RADIX_BITS, end_bit - current_bit);
```

Reply:
This is genius

Reply:
> This is genius

It's just fighting C++ with C++.